test/GithubApi.Authentication.test.js

```diff
@@ -0,0 +1,27 @@
+const agent = require('superagent-promise')(require('superagent'), Promise);
+const statusCode = require('http-status-codes');
+const { expect } = require('chai');
+
+const urlBase = 'https://api.github.com';
+const githubUserName = 'afranco7';
+const repository = 'psl-workshop-api-testing-js';
+
+describe('Github Api Test', () => {
+  describe('Authentication', () => {
+    it('Via OAuth2 Tokens by Header', () =>
+      agent.get(`${urlBase}/repos/${githubUserName}/${repository}`)
+        .auth('token', process.env.ACCESS_TOKEN)
+        .then((response) => {
+          expect(response.status).to.equal(statusCode.OK);
+          expect(response.body.description).equal('This is a Workshop about Api Testing in JavaScript');
+        }));
+  });
+});
+
+it('Via OAuth2 Tokens by parameter', () =>
```

Comment:
agrega este it dentro del describe (Todos it debería estar dentro de un describe)